src/Common/hooks/useFilters.tsx

```diff
@@ -266,9 +280,12 @@ const getFiltersCacheKey = () => `filters--${window.location.pathname}`;
 const getFiltersCache = () => {
   return JSON.parse(localStorage.getItem(getFiltersCacheKey()) || "{}");
 };
-const updateFiltersCache = (cache: Record<string, unknown>) => {
+const updateFiltersCache = (
+  cache: Record<string, unknown>,
+  cacheBlacklist: string[]
+) => {
   const result = { ...cache };
-  for (const param of FILTERS_CACHE_BLACKLIST) {
+  for (const param of FILTERS_CACHE_BLACKLIST.concat(cacheBlacklist)) {
```

Comment:
Skip multiple concats since the caller would already passes the concated final list

```suggestion
  for (const param of cacheBlacklist) {
```

Reply:
didnt understand this 
shouldnt we concat both the arrays

Reply:
you have to concat both arrays, but you need not concat it two times. you would already be passing the concated items to the args, and hence you need not perform concat again here.